### PR TITLE
fix issue #1566

### PR DESCRIPTION
**Pull request by youngsterxyf**
R.T. #1566

### CODE REVIEW

File: logs/console.go [hunk 1]
```diff
@@ -69,10 +70,14 @@ func (c *consoleWriter) Init(jsonconfig string) error {
 }
 
 // WriteMsg write message in console.
-func (c *consoleWriter) WriteMsg(msg string, level int) error {
+func (c *consoleWriter) WriteMsg(when time.Time, msg string, level int) error {
 	if level > c.Level {
 		return nil
 	}
+
+	logTimeStr := formatLogTime(when)
```
Comment (JessonChan) [inline logs/console.go]: log package will write the now time,your log message will be like this  
2016-01-23 23:23:00 2016-01-23 23:23:00 ....

Comment (youngsterxyf) [inline logs/console.go]: I set log.New's flag parameter to zero, it should not write the now time

Comment (JessonChan) [inline logs/console.go]: it's ok.

File: logs/log.go [hunk 2]
```diff
@@ -261,7 +264,7 @@ func (bl *BeeLogger) Critical(format string, v ...interface{}) {
 		return
 	}
 	msg := fmt.Sprintf("[C] "+format, v...)
-	bl.writeMsg(LevelCritical, msg)
+	bl.writeMsg(LevelCritical, msg, time.Now())
```
Comment (JessonChan) [inline logs/log.go]: you should not pass a time.Tim arg into func writeMsg,but new a time.Time object in the writeMsg func

Comment (youngsterxyf) [inline logs/log.go]: Yes，it may be better。

File: logs/conn.go [hunk 3]
```diff
@@ -62,6 +63,10 @@ func (c *connWriter) WriteMsg(msg string, level int) error {
 	if c.ReconnectOnMsg {
 		defer c.innerWriter.Close()
 	}
+
+	logTimeStr := formatLogTime(when)
```
Comment (JessonChan) [inline logs/conn.go]: no more logTimeStr var here

File: logs/console.go [hunk 4]
```diff
@@ -69,10 +70,14 @@ func (c *consoleWriter) Init(jsonconfig string) error {
...
+	msg = logTimeStr + msg
```
Comment (JessonChan) [inline logs/console.go]: remove var logTimeStr

File: logs/file.go [hunk 5]
```diff
@@ -114,50 +114,15 @@ func (w *fileLogWriter) needRotate(size int, day int) bool {
...
-	buf[15] = byte('0' + mi - t*10)
-	buf[16] = ':'
-	t = s / 10
-	buf[17] = byte('0' + t)
-	buf[18] = byte('0' + s - t*10)
-	buf[19] = ' '
-	msg = string(buf[0:]) + msg + "\n"
+	// now := time.Now()
+	d := when.Day()
+	logTimeStr := formatLogTime(when)
```
Comment (JessonChan) [inline logs/file.go]: same commnet

File: logs/console.go (hunk 4 above)
Comment (youngsterxyf) [inline logs/console.go]: OK, I know, performance first, Thanks

**Comment by JessonChan** [comment]
LGTM now .@astaxie

**Pull request merged**